hc/accounts/tests/test_login.py

```diff
@@ -18,14 +17,14 @@ def test_it_sends_link(self):
 
         r = self.client.post("/accounts/login/", form)
         assert r.status_code == 302
-
-        ### Assert that a user was created
-
+        ### **Assert that a user was created
+        final_list = len(User.objects.all())
+        self.assertEqual(1, final_list)
```

Comment:
While this asserts that the list queryset for users has one object in it, there is no telling if the object represents the user that was created. Add more statements to assert that.